tests/storage/cdi_import/test_import_http.py

```diff
@@ -542,6 +542,7 @@ def test_blank_disk_import_validate_status(data_volume_multi_storage_scope_funct
     ],
 )
 def test_vmi_image_size(
+    skip_test_if_gcp_sc,
```

Comment:
Why do we want to skip this test?

Reply:
we should not have such programatic skips